konfera/migrations/0019_talk_language.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2016-12-07 23:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('konfera', '0018_auto_20161203_2305'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='talk',
+            name='language',
+            field=models.CharField(choices=[('SK', 'Slovak'), ('EN', 'English')], default='EN', max_length=2),
```

Comment:
I don't see Czech here. Maybe you created the migrations before you add the language to the list.